tokens/models/tokens/arbitrum/tokens_arbitrum_erc20.sql

```diff
@@ -79,4 +79,9 @@ FROM (VALUES
     , (0x323665443cef804a3b5206103304bd4872ea4253, 'USDV', 6)
     , (0x966570a84709d693463cdd69dcadb0121b2c9d26, 'taoUSD', 18)
     , (0x993614e1c8c9c5abe49462ce5702431978fd767f, 'S*ETH', 18)
+    , (0xaf88d065e77c8cc2239327c5edb3a432268e5831, 'USDC', 6) -- Native
```

Comment:
i'm still a bit confused on this approach here. these stablecoins already exist in `tokens.erc20`:
```
select
  *
from
  tokens.erc20
where
  contract_address = 0xaf88d065e77c8cc2239327c5edb3a432268e5831
```

keep in mind that erc20 token metadata now combines an automated source + these static files to handle outlier, long tail tokens missing. i would validate all of these already exist in the table, then remove from this PR.

Reply:
> keep in mind that erc20 token metadata now combines an automated source

Wow did I ever jump the gun on this. This hurts my soul.